AR 2023

cjyabraham opened the pull request:
[Dev instance](https://pr-821-cncfci.pantheonsite.io/reports/cncf-annual-report-2023/)

thetwopct commented:
Great stuff, looks good. A few things I found below, I actioned some other things directly. 

1) The hero looks a bit odd on larger monitors

![Screenshot-2024-01-09 --14 20 42](https://github.com/cncf/cncf.io/assets/10615884/0dc31521-e908-4f08-b1ef-05d7e57924c9)

2) The Social Share still has Twitter in it - not X. The code could be replaced with the social share component as other recent reports also have done.

3) The John Furrier tweet is really hard to read on mobile. Suggested to Gareth that the design changes but was not actioned. Even taking a screenshot from Twitter would make it more readable on mobile as the line length is shorter.

4) Humans of Cloud Native "Nominate" button - I would strongly advocate for that to not be a button as it's a mailto: link. It should just be written in the paragraph above so it's obvious it's an email address. 

5) The large picture in Community Awards is related to the chop wood but is posted next to spotlight awards, could make more sense with it moved up?

6) The footer image seems to wander off a bit on wide screens and on mobile it looks cut off, not sure if intentional? If it is, doesnt look great.

![Screenshot-2024-01-09 --14 59 43](https://github.com/cncf/cncf.io/assets/10615884/da48af9f-f7fa-4848-b966-da450571f634)

7) We normally have a share panel at the bottom of the reports

8) On mobile - Kubernetes Community Days stats - the lines between them have something wrong, and the text seems a little large for the screen size. 

9) On mobile - New Projects - looks like the icon+stat could wrap on two a new line as quite squashed as is

cjyabraham commented:
1. Fixed
2. Fixed
3. Took a screenshot from Twitter would make it more readable on mobile
4. Fixed
5. I think it's ok where it is because it anchors the whole section.
6. I fixed the wide-screen issue but didn't do anything about mobile... wasn't sure quite what to do there.
7. The [Kubecon NA report](https://www.cncf.io/reports/kubecon-cloudnativecon-north-america-2023/) didn't have a share at the bottom and I thought it would work better to have the Kubecon ad instead as we don't have that anywhere else. 
8. The text size is consistent with other such blocks on mobile. I fixed the padding for the borders.
9. Fixed.
